infra/helper.py

```diff
@@ -283,6 +293,27 @@ def _env_to_docker_args(env_list):
   return sum([['-e', v] for v in env_list], [])
 
 
+def _workdir_from_dockerfile(dockerfile_path):
+  """Parse WORKDIR from the Dockerfile."""
+  WORKDIR_REGEX = re.compile(r'\s*WORKDIR\s*([^\s]+)')
+
+  with open(dockerfile_path) as f:
+    lines = f.readlines()
+
+  for line in reversed(lines):  # reversed to get last WORKDIR.
+    match = re.match(WORKDIR_REGEX, line)
+    if match:
+      workdir = match.group(1)
+      workdir = workdir.replace('$SRC', '/src')
+
+      if not os.path.isabs(workdir):
+        workdir = os.path.join('/src', workdir)
+
+      return workdir
+
+  return None
```

Comment:
Won't we error out here in caller ?, should we return /src/$project_name as default ?